views.py

```diff
@@ -340,4 +340,5 @@ async def download_configs_zip(request):
 async def get_gui_config(request):
     with open(gui_config_path) as yaml_config:
         json_config = yaml.safe_load(yaml_config)
+        json_config["coeff_dir"] = join(request.app["coeff_dir"], '')  # append folder separator at the end
```

Comment:
I think it would be easier to handle if you made a new dict, where the config is one value and the coeff_dir is another:
```
json_to_send = {"config": json_config, "coeff_dir": join(request.app["coeff_dir"], '')} 
```
Then the camilla config stays as usual, should be less confusing on the other end.